src/main/java/school/lemon/changerequest/java/introduction/hw1/Task4.java

```diff
@@ -4,6 +4,21 @@
 public class Task4 {
 
     public static double getSumOfGreatest(double a, double b, double c) {
-        return 0;
+        Double[] array = {a,b,c};
```

Comment:
Overcomplicated. You have only 3 numbers, there is no need to implement sorting.